package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "vui-table",
+  "version": "1.0.0",
+  "description": "Mixins and CSS for applying table styles",
+  "scripts": {
+    "autoprefix": "postcss -c postcss.config.json",
+    "autoprefix:test": "postcss -c ./test/postcss.config.json",
+    "clean": "rimraf *.css && rimraf ./test/*.css",
+    "build:sass:test": "node-sass --output-style expanded ./test/test.scss ./test/test.css",
+    "postbuild:sass:test": "npm run autoprefix:test",
+    "build:sass": "node-sass --output-style expanded table.css.scss table.css",
+    "postbuild:sass": "npm run autoprefix",
+    "prebuild": "npm run clean",
+    "build": "npm run build:sass && npm run build:sass:test",
+    "perceptual-diff": "percy snapshot test/",
```

Comment:
I'm still waiting for finance to put the purchase through, will let the team know when it's done.